web/src/components/__tests__/PoisDesktop.spec.tsx

```diff
@@ -55,7 +55,6 @@ describe('PoisDesktop', () => {
     expect(queryByText('pois:distanceKilometre')).toBeTruthy()
     expect(queryByText(singlePoi.location.address!)).toBeTruthy()
     expect(queryByText(singlePoi.content)).toBeTruthy()
-    expect(queryByText('pois:makeAppointment')).toBeTruthy()
```

Comment:
🤔 Why is this not truthy anymore/why did you remove it?

Reply:
Because its not at Poi anymore I moved the appointmentLink to OpeningHours component like native. 🙂